spec/javascripts/components/leaflet_iiif_spec.js.jsx

```diff
@@ -0,0 +1,13 @@
+/* globals LeafletIiif */
+
+describe('LeafletIiif', () => {
+  it('sets up the iiif layer', () => {
+    const component = TestUtils.renderIntoDocument(
+      <LeafletIiif
+        iiifImage="https://stacks.stanford.edu/image/iiif/hg676jb4964%2F0380_796-44/info.json"
+      />,
+    );
```

Comment:
When I run eslint locally with the included .eslintrc, I don't actually get a complaint about this.